lib/LaTeXML/Package/orcidlink.sty.ltxml

```diff
@@ -0,0 +1,41 @@
+# -*- mode: Perl -*-
+# /=====================================================================\ #
+# |  paralist                                                           | #
```

Comment:
[minor] You may want to change the docstring to orcidlink here.

Reply:
Ahem.